strangerstudios/pmpro-payfast
# Fix meta save

### Changes proposed in this Pull Request:

Remove callback functions `pmpro_payfast_before_send_to_payfast_save_data` and `pmpro_payfast_before_send_to_payfast_save_data` to avoid possible interference for saving meta at checkout.



Resolves #97 .

### How to test the changes in this Pull Request:

1. Apply patch.
2. Check out for a paid level with custom user fields that can contain an array, e.g. `chec`.koutbox_grouped`, `select2
3. Confirm that custom user field data was saved to user meta and displayed in admin notification email.

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
